proto/Cargo.toml

```diff
@@ -15,14 +15,12 @@ build = "build.rs"
 [features]
 default = ["protobuf-codec"]
 protobuf-codec = ["protobuf-build/protobuf-codec"]
-prost-codec = ["prost", "prost-derive", "bytes", "lazy_static", "protobuf-build/prost-codec"]
+prost-codec = ["protobuf-build/prost-codec", "prost", "lazy_static"]
```

Comment:
Interesting. Isn't prost requires bytes dependency?

Reply:
prost requires bytes dependency, but raft does not explicitly use the methods in Buf and BufMut traits

Reply:
Prost 0.6 fixed that requirement, it's only required with 0.5 unless you use Buf/BufMut, etc.